src/commands/lpop.json

```
@@ -1,6 +1,6 @@
 {
     "LPOP": {
-        "summary": "Returns the first elements in a list after removing it. Deletes the list if the last element was popped.",
+        "summary": "Returns the first elements in a list after removing it or up to count elements if the optional count argument is provided. Deletes the list if the last element was popped.",
```

What do you think about a simpler description like "one or more elements"?

```suggestion
        "summary": "Returns and removes one or more elements from the beginning of a list. Deletes the list if the last element was popped.",
```

---

Address comment, Thanks